### DISCUSSION TITLE
Powered by SQLPage section in website / doco

### DISCUSSION THREAD

**Discussion by accforgithubtest** [sqlpage/SQLPage#173] [Ideas]
I came across [this repo](https://github.com/DSMejantel/Ecole_inclusive) browsing thru the issues list, and just thought that it would be useful to have a section somewhere on the website / doco, that links to other openly available SQLPage resources, especially fully featured ones like above.

I think it would be useful both as a positive advertisement for how this awesome application is used, as well as sort of a reference to code.

**Comment by DSMejantel** [comment]
Hi,
I'a a newbee on github and Ecole Inclusive is my first appli.
I can do that with the competences and the kidness of lovasoa.
I'm agree with you, it can be a good idea to share the experiences with SQLpage
I think, i will open discussions on my github too...

**Reply by DSMejantel** [reply]
Hello,
discussions are open on École Inclusive.
